### ISSUE TITLE
Create a check to measure the page weight of the homepage

### ISSUE THREAD

**Issue by seanhamlin** [seanhamlin/site-audit#42]
Would be nice to breakdown the requests by type, e.g. HTML, images, JS, CSS etc

**Comment by seanhamlin** [comment]
Doing some investigation, integration with https://github.com/macbre/phantomas would be awesome. It can produce JSON like:

```
$ phantomas https://www.[DOMAIN].com/ --reporter=json:pretty
{
  "generator": "phantomas v1.17.0",
  "url": "https://www.[DOMAIN].com/",
  "metrics": {
    "requests": 23,
    "gzipRequests": 8,
    "postRequests": 0,
    "httpsRequests": 23,
    "notFound": 0,
    "bodySize": 1050511,
    "contentLength": 1628148,
    "httpTrafficCompleted": 2258,
    "timeToFirstByte": 1340,
    "timeToLastByte": 1352,
    "ajaxRequests": 0,
    "htmlCount": 1,
    "htmlSize": 9859,
    "cssCount": 3,
    "cssSize": 24399,
    "jsCount": 4,
    "jsSize": 175570,
    "jsonCount": 0,
    "jsonSize": 0,
    "imageCount": 14,
    "imageSize": 1327908,
    "videoCount": 0,
    "videoSize": 0,
    "webfontCount": 0,
    "webfontSize": 0,
    "base64Count": 0,
    "base64Size": 0,
    "otherCount": 1,
    "otherSize": 90412,
    "cacheHits": 22,
    "cacheMisses": 1,
    "cachePasses": 0,
    "cachingNotSpecified": 0,
    "cachingTooShort": 0,
    "cachingDisabled": 0,
    "oldCachingHeaders": 23,
    "consoleMessages": 0,
    "cookiesSent": 0,
    "cookiesRecv": 0,
    "domainsWithCookies": 0,
    "documentCookiesLength": 8,
    "documentCookiesCount": 1,
    "documentHeight": 1981,
    "commentsSize": 324,
    "whiteSpacesSize": 2153,
    "DOMelementsCount": 526,
    "DOMelementMaxDepth": 19,
    "nodesWithInlineCSS": 15,
    "imagesScaledDown": 0,
    "imagesWithoutDimensions": 2,
    "DOMidDuplicated": 0,
    "hiddenContentSize": 969,
    "hiddenImages": 1,
    "DOMmutationsInserts": 21,
    "DOMmutationsRemoves": 10,
    "DOMmutationsAttributes": 240,
    "DOMqueries": 146,
    "DOMqueriesWithoutResults": 7,
    "DOMqueriesById": 7,
    "DOMqueriesByClassName": 17,
    "DOMqueriesByTagName": 121,
    "DOMqueriesByQuerySelectorAll": 1,
    "DOMinserts": 23,
    "DOMqueriesDuplicated": 12,
    "DOMqueriesAvoidable": 26,
    "domains": 1,
    "maxRequestsPerDomain": 23,
    "medianRequestsPerDomain": 23,
    "eventsBound": 118,
    "eventsDispatched": 0,
    "eventsScrollBound": 1,
    "globalVariables": 4,
    "globalVariablesFalsy": 0,
    "headersCount": 473,
    "headersSentCount": 68,
    "headersRecvCount": 405,
    "headersSize": 13967,
    "headersSentSize": 2620,
    "headersRecvSize": 11347,
    "headersBiggerThanContent": 1,
    "jQueryVersion": "1.9.1",
    "jQueryVersionsLoaded": 1,
    "jQueryOnDOMReadyFunctions": 4,
    "jQueryWindowOnLoadFunctions": 0,
    "jQuerySizzleCalls": 51,
    "jQueryEventTriggers": 5,
    "jQueryDOMReads": 16,
    "jQueryDOMWrites": 95,
    "jQueryDOMWriteReadSwitches": 13,
    "documentWriteCalls": 0,
    "evalCalls": 0,
    "jsErrors": 0,
    "closedConnections": 0,
    "lazyLoadableImagesBelowTheFold": 3,
    "localStorageEntries": 0,
    "redirects": 0,
    "redirectsTime": 0,
    "repaints": 0,
    "firstPaint": 0,
    "requestsToFirstPaint": 0,
    "domainsToFirstPaint": 0,
    "requestsToDomContentLoaded": 8,
    "domainsToDomContentLoaded": 1,
    "requestsToDomComplete": 23,
    "domainsToDomComplete": 1,
    "assetsNotGzipped": 2,
    "assetsWithQueryString": 0,
    "assetsWithCookies": 0,
    "smallImages": 3,
    "smallCssFiles": 1,
    "smallJsFiles": 1,
    "multipleRequests": 0,
    "timeToFirstCss": 1382,
    "timeToFirstJs": 1462,
    "timeToFirstImage": 2112,
    "domInteractive": 567,
    "domContentLoaded": 564,
    "domContentLoadedEnd": 761,
    "domComplete": 911,
    "timeBackend": 60,
    "timeFrontend": 40,
    "requestsWithTimeout": 0,
    "statusCodesTrail": "200",
    "windowAlerts": 0,
    "windowConfirms": 0,
    "windowPrompts": 0,
    "bodyHTMLSize": 34657,
    "iframesCount": 0,
    "smallestResponse": 264,
    "biggestResponse": 332377,
    "fastestResponse": 39,
    "slowestResponse": 1352,
    "smallestLatency": 38,
    "biggestLatency": 1340,
    "medianResponse": 212,
    "medianLatency": 211
  },
  ...
```

And much more

**Comment by seanhamlin** [comment]
Fixed in af45fbbcb4e409ac0f3b8fb3481846b55e5bcf1c
